lib/ts/recipe/emailpassword/components/themes/default/emailVerification/emailVerificationScreen/sendVerifyEmail.tsx

```diff
@@ -0,0 +1,134 @@
+/* Copyright (c) 2020, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Imports.
+ */
+import React, { PureComponent } from "react";
+import StyleContext from "../../../../styles/styleContext";
+
+import { SendVerifyEmailThemeProps, SendVerifyEmailThemeState } from "../../../../../types";
+
+/** @jsx jsx */
+import { jsx } from "@emotion/react";
+import EmailIcon from "../../../../../assets/emailLargeIcon";
+import ArrowRightIcon from "../../../../../assets/arrowRightIcon";
+import { API_RESPONSE_STATUS, SEND_VERIFY_EMAIL_STATUS } from "../../../../../constants";
+import { SOMETHING_WENT_WRONG_ERROR } from "../../../../../../../constants";
+
+/*
+ * Component.
+ */
+
+export default class SendVerifyEmail extends PureComponent<SendVerifyEmailThemeProps, SendVerifyEmailThemeState> {
+    static contextType = StyleContext;
+    /*
+     * Constructor.
+     */
+    constructor(props: SendVerifyEmailThemeProps) {
+        super(props);
+        this.state = {
+            status: SEND_VERIFY_EMAIL_STATUS.READY
+        };
+    }
+
+    /*
+     * Methods.
+     */
+
+    onSuccess = (): void => {
+        this.setState(() => ({
+            status: SEND_VERIFY_EMAIL_STATUS.SUCCESS
+        }));
+        if (this.props.onSuccess !== undefined) {
+            this.props.onSuccess();
+        }
+    };
+
+    sendEmail = async (): Promise<void> => {
+        const { callAPI } = this.props;
```

Comment:
Are you sure that `callAPI` is a good name for this? I don't think we have discussed the props given from the feature component to the theme component

Reply:
This is the name in use already for Sign Up / Sign In and reset password.
See https://github.com/supertokens/supertokens-auth-react/blob/42492bce51255148572f1b6742d7e72a00c2630b/lib/ts/recipe/emailpassword/types.ts#L340